include/ginkgo/core/solver/solver_base.hpp

```diff
@@ -70,17 +70,23 @@ enum class initial_guess_mode {
 };
 
 
+namespace multigrid {
```

Comment:
inconsistent namespace with the other `multigrid.hpp` include, I guess you already fixed it